agent/host.go

```diff
@@ -134,5 +163,39 @@ func createRootNode(g *graph.Graph) (*graph.Node, error) {
 		m.SetField("VirtualizationRole", hostInfo.VirtualizationRole)
 	}
 
+	var lshwMap map[string]interface{}
+	lshw, err := exec.Command("lshw", "-quiet", "-json").Output()
+	if err == nil {
+		err = json.Unmarshal(lshw, &lshwMap)
+		if err == nil {
+			parseLshw(lshwMap)
+			m.SetField("Hardware", lshwMap)
+		}
+	} else {
```

Comment:
maybe we should always populate "CPU" field regardless of if we managed to extract data via lshw

Reply:
If we get more info from lshw, what would be the need of having duplicated info ?

Reply:
what i don't like is that the fields (starting from top-level node -  "CPU") change, as a result is i write a Gremlin Query (such as to retrieve nodes with X86 architecture) - the query will not hold. One alternative (to what i proposed above) is to use lshw to enrich the "CPU" metadata node

Reply:
That is opening an interesting point. How much the structure should be predictable ?
Does this means it would be better to "reduce" the number of reported information or format it into predefined structures ? @lebauce @safchain ?

Reply:
I would prefer to have predefined structure so that we guaranty the Skydive output, better to do gremlin request, etc.